### PR TITLE
solution

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
[DEMO LINK](https://nikitakarbazin.github.io/react_sum/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-    {/* And remove comment lines :) */}
-  </>
+  <div className="block">
+    <Sum a={2} b={3} />
+    <Sum a={-5} b={5} />
+    <Sum a={10} b={0} />
+    <Sum a={0} b={5} />
+    <Sum a={0} b={0} />
+  </div>
 );
```
Comment (stanislav-topikha) [inline src/App.jsx:14]:
```suggestion
  <div className="block">
    <Sum a={2} b={3} />
    <Sum a={-5} b={5} />
    <Sum a={10} />
    <Sum b={5} />
    <Sum />
  </div>
);
```

File: src/components/Sum.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Sum.propsTypes = {
+  a: PropsTypes.number,
+  b: PropsTypes.number,
+};
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```
Comment (stanislav-topikha) [inline src/components/Sum.jsx:20]: I guess you don't really need this, if you have this 
image

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
It works)
BUT ⏬ 
image

**Review by stanislav-topikha — APPROVED** [review]

File: src/components/Sum.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropsTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => {
+  const text = `Sum of ${a} and ${b} is ${a + b}`;
+
+  return (
+    <p>{text}</p>
```
Comment (Olexandr6) [inline src/components/Sum.jsx:8]: Why do you need a variable here?
```suggestion
    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**